src/background/services/wallet/handlers/importLedger.ts

```diff
@@ -105,10 +117,7 @@ export class ImportLedgerHandler implements HandlerType {
       });
     }
 
-    const numberOfAccountsToCreate =
-      secretType === SecretType.LedgerLive ? pubKeys?.length : undefined;
-
-    await this.#addAccounts(id, numberOfAccountsToCreate);
+    await this.#addAccounts(id, numberOfAccountsToCreate || 3);
```

Comment:
Please add a check to make sure that the `numberOfAccountsToCreate` is <= than `pubKeys?.length` when we don't have the xPub.